integration-tests/src/test/java/tachyon/underfs/oss/OSSUnderStorageCluster.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.underfs.oss;
+
+import java.io.IOException;
+import java.util.UUID;
+
+import tachyon.Constants;
+import tachyon.conf.TachyonConf;
+import tachyon.underfs.UnderFileSystem;
+import tachyon.underfs.UnderFileSystemCluster;
+
+public class OSSUnderStorageCluster extends UnderFileSystemCluster {
+
+  private static final String INTEGRATION_OSS_BUCKET = "ossBucket";
+
+  private String mOSSBucket;
+
+  public OSSUnderStorageCluster(String baseDir, TachyonConf tachyonConf) {
+    super(baseDir, tachyonConf);
+    String ossAccessId = System.getProperty(Constants.OSS_ACCESS_KEY);
```

Comment:
line 34 to line 39 are unnecessary, as `tachyonConf` will pick up those properties automatically. 
You can take a look at the latest `S3UnderStorageCluster`